src/gsi/rapidrefresh_cldsurf_mod.f90

```diff
@@ -447,6 +458,10 @@ subroutine init_rapidrefresh_cldsurf
     corp_howv           = 0.42_r_kind                 ! 0.42 meters (default)
     hwllp_howv          = 170000.0_r_kind             ! 170,000.0 meters (170km as default for 3DRTMA, 50km is used in 2DRTMA)
     i_howv_3dda         = 0                           ! no analysis of significant wave height (howv) in 3D analysis (default)
+    corp_gust           = -3.00_r_kind                ! default to use the value (3.0 m/s) set in m_berror_stats_reg.f90
```

Comment:
The comment seems not explaining why they are first initialized to negative values.

Reply:
@TingLei-NOAA 
Hi Ting,
The negative value means to use the default value which has been hardwired in GSI code (subroutine berror_read_wgt_reg). I will add more explanation in the comment to make it more clear.
Thanks,
-Gang